ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -269,55 +279,47 @@ func (p *LocalProject) convertOverrideCompose() error {
 	return nil
 }
 
-// Write writes the compose data to a local compose file. The output filename is stored on the project
+// Write writes the compose data to a local compose file. The output filename
+// is stored on the project
 func (p *LocalProject) Write() error {
-	if err := writeFile(p.LocalOutFileName(), p.composeBytes[baseComposeIndex]); err != nil {
+	// write compose file
+	if err := p.writeFile(p.LocalOutFileName(), p.composeBytes[baseComposeIndex], false); err != nil {
 		return err
 	}
 
-	if err := writeNewFile(p.OverrideFileName(), p.composeBytes[overrideComposeIndex]); err != nil {
+	// write override file
+	if err := p.writeFile(p.OverrideFileName(), p.composeBytes[overrideComposeIndex], true); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-// writeFile writes the content to a filename. If the file already exists, then prompts the user for overwrite.
-func writeFile(filename string, content []byte) error {
-	f, err := openFile(filename)
-	defer f.Close()
+// writeFile writes the content to a filename. If the file already exists, user is
+// prompted to overwrite unless force flag specified.
+func (p *LocalProject) writeFile(filename string, content []byte, skipOverwrite bool) error {
```

Comment:
What do you think about splitting the function into two instead of having a boolean parameter? https://martinfowler.com/bliki/FlagArgument.html

`writeFile` and `forceWriteFile` or something like that

Reply:
that's what it was before but it was not very DRY (see: writeNewFile func). I went this way also bc it made it easier to test whether or not the overwrite function was being called: https://github.com/aws/amazon-ecs-cli/pull/907/files#diff-589e4b27e2ac8134f1c4c268a5a4f3c4R314-R318